### PR TITLE
GIVCAMP-187 | Fake Chatbot PoC

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- "Fake it till you make it" chatbot interaction
- It allows user to make a choice and conditionally display content at the end

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

Note: This is held together by duct tape 🤣 But it is just a POC so if they like it, I'll clean it up for sure.
1. https://deploy-preview-114--giving-campaign.netlify.app/stories/poc-ai-chatbot-story
2. Scroll to the "chatbot" interface and give it a try

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-187

### CODE REVIEW

**Comment by sherakama** [comment]
I lol'd but I am also very curious and excited about the opportunity to build out a chatbot.

**Comment by yvonnetangsu** [comment]
> I lol'd but I am also very curious and excited about the opportunity to build out a chatbot.

Hahaha SODA would like a conversation with you @sherakama 🤣

File: app/[...slug]/page.tsx [hunk 1]
```diff
@@ -65,8 +65,9 @@ export async function generateMetadata({ params }: { params: ParamsType }): Prom
 export default async function Page({ params }: { params: ParamsType }) {
   const { data } = await getStoryData(params);
   const bridgeOptions = { resolveRelations };
+  let slug: string = params.slug ? params.slug.join('/') : '';
 
   return (
-    <StoryblokStory story={data.story} bridgeOptions={bridgeOptions} />
+    <StoryblokStory story={data.story} bridgeOptions={bridgeOptions} slug={slug} />
```
Comment (yvonnetangsu) [inline app/[...slug]/page.tsx:71]: In the future we might want to access the slug in in the stories so passing this in. I've tested that this works.

Also, this route [...slug] is for non-homepage so I'm removing the "home" condition.

**Comment by yvonnetangsu** [comment]
I'm going to merge the fake bot in since this is a total PoC 🤖

**Pull request merged**